homeassistant/helpers/entity_platform.py

```diff
@@ -487,15 +487,6 @@ async def _async_add_entity(  # noqa: C901
             self._get_parallel_updates_semaphore(hasattr(entity, "update")),
         )
 
-        # Update properties before we generate the entity_id
-        if update_before_add:
```

Comment:
It was up here because the data can be used for creating entity/device registry entries

Reply:
Right, we want a computed name to be registered as the `original_name`, and so on.